apps/system/utils/utils_heapinfo.c

```diff
@@ -291,6 +284,13 @@ int utils_heapinfo(int argc, char **args)
 	}
 	close(heapinfo_fd);
 
+#ifdef CONFIG_BUILD_PROTECTED
```

Comment:
Why did you move this? For clarifying which heap to see, it is better to print before calling parse heap(`ioctl(heapinfo_fd, MMINFOIOC_PARSE, (int)&options)`).

Reply:
Sure.. I had something else in mind when i moved this down. 
But, maybe its not required. I will revert it.